content/en/blog/_posts/2023-04-11-topology-spread-features.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: blog
+title: "Kubernetes 1.27: More fine-grained pod topology spread policies reached beta"
+date: 2023-04-11
+slug: fine-grained-pod-topology-spread-features-beta
+evergreen: true
+---
+
+**Authors:** [Alex Wang](https://github.com/denkensk)(Shopee), [Kante Yin](https://github.com/kerthcet)(DaoCloud), [Kensei Nakada](https://github.com/sanposhiho)(Mercari)
+
+In Kubernetes v1.19, [Pod Topology Spread Constraints](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/) went to GA.
+It is the feature to control how Pods are spread to each failure-domain (regions, zones, nodes etc).
+
+As time passes, we've got further feedbacks from users,
+and we're actively working on improving the Topology Spread via three KEPs from v1.25.
+All of these features have reached beta in Kubernetes v1.27 and been enabled by default.
+
+This blog post is going to introduce each feature and the usecase/issue behind them.
+
+## KEP-3022: min domains in Pod Topology Spread
+
+Pod Topology Spread has the `maxSkew` parameter to define the degree to which Pods may be unevenly distributed. 
+
+But, there wasn't a way to control the number of domains over which we should spread. 
```

Comment:
I would add a sentence along these lines:
kube-scheduler can only operate based on the Nodes that currently exist in a cluster. However, in a cloud environment, the cluster-autoscaler can add or remove Nodes based on demand. As a result, users need to
control the spread over topology domains that might not have any Nodes at a given time.